packages/react/src/primitives/Alert/__tests__/Alert.test.tsx

```diff
@@ -44,10 +44,13 @@ describe('Alert: ', () => {
       </Alert>
     );
 
-    const alert = await screen.findByText('Test heading');
-    expect(alert.nodeName).toBe('H6');
+    const alertHeading = await screen.findByText('Test heading');
+    expect(alertHeading.nodeName).toBe('DIV');
     expect(
-      alert.parentElement.parentElement.parentElement.classList.contains(
+      alertHeading.classList.contains(ComponentClassNames.AlertHeading)
+    ).toBe(true);
+    expect(
+      alertHeading.parentElement.parentElement.classList.contains(
```

Comment:
nit: These tests look brittle. We could find some time to refactor it by querying elements in a better manner